src/common/darktable.c

```diff
@@ -1655,11 +1669,24 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
   }
   free(config_info);
 
-  // last but not least construct the popup that asks the user about
-  // images whose xmp files are newer than the db entry
-  if(init_gui && changed_xmp_files)
+  if(init_gui)
   {
-    dt_control_crawler_show_image_list(changed_xmp_files);
+    // start the thumbnail backthread only if using the default db
+    const gboolean backthumb =
+        !dbfilename_from_command
```

Comment:
On my side I have multiple DB which I select at launch time. My test DB is in this case and so the crawler was not started. Hence my question, why this check? Do you foresee issues? I bet we are many to have multiple DB, so we should probably find a solution to this.

Reply:
I was not sure when implementing it. And we certainly have inconsistencies with changed timestamp between databases and different iophash.

Let me think about it a bit more.

Reply:
When starting from the command line one should change the DB and cache location. So what could be the inconsistencies?

Reply:
Not relating database and mipmap cache location?

Reply:
I understand, this is a bad usage permitted by the current dt command line. Should be fixed there.

I had this in mind since some time: #15394

Reply:
nope. i forgot that and it seems fully safe for me.

Updated code working for other "config whatever way" database, some code simplifications.